modules/op-resolver-hub.adoc

```diff
@@ -0,0 +1,138 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-hub-specify_{context}"]
+= Specifying a remote pipeline or task using the Hub resolver
+
+When creating a pipeline run, you can specify a remote pipeline from {artifact-hub} or {tekton-hub}. When creating a pipeline or a task run, you can specify a remote task from {artifact-hub} or {tekton-hub}.
+
+.Procedure
+
+* To specify a remote pipeline or task from {artifact-hub} or {tekton-hub}, use the following reference format in the `pipelineRef` or `taskRef` spec:
++
+[source,yaml]
+----
+# ...
+  resolver: hub
+  params:
+  - name: catalog
+    value: <catalog>
+  - name: type
+    value: <catalog-type>
+  - name: kind
+    value: [pipeline|task]
+  - name: name
+    value: <resource-name>
+  - name: version
+    value: <resource-version>
+# ...
+----
++
+.Supported parameters for the hub resolver
+|===
+| Parameter | Description | Example value
+
+| `catalog`
+| The catalog for pulling the resource.
+| Default:  `tekton-catalog-tasks` (for the `task` kind);  `tekton-catalog-pipelines` (for the `pipeline` kind).
+
+| `type`
+| The type of the catalog for pulling the resource. Either `artifact` for {artifact-hub} or `tekton` for {tekton-hub}.
+| Default:  `artifact`
+
+| `kind`
+| Either `task` or `pipeline`.
+| Default: `task`
+
+| `name`
+| The name of the task or pipeline to fetch from the hub.
+| `golang-build`
+
+| `version`
+| The version of the task or pipeline to fetch from the hub. You must use quotes (`"`) around the number.
+| `"0.5.0"`
+|===
++
+If the pipeline or task requires additional parameters, provide these parameters in `params`.
```

Comment:
It still looks weird to me that this is a procedure with a bunch of reference examples, and the indentation is odd:
![image](https://github.com/openshift/openshift-docs/assets/40172997/18de600d-9fee-4bbb-885a-63873e7ef891)

I'd recommend adding an `.Examples` heading or something before these examples but will defer to @bergerhoffer on whether these are acceptable as is.

Reply:
agreed in Slack to keep as is